README.md

```diff
@@ -1,126 +1,164 @@
-| hadoop\_namenode\_dfs\_nameservices | dfs.nameservices | mycluster |
-| hadoop\_namenode\_master | hostname of master | "" |
-| hadoop\_namenode\_slave | hostname of slave | "" |
-| hadoop\_namenode\_enable\_ha | enable HA configuration | false |
-
-
-Dependencies
-------------
-
-dict2xml is bundled in the role, with modifications, mostly removing six dependency. dict2xml is available at https://github.com/delfick/python-dict2xml.
-
-Example Playbook
-----------------
-
-  - hosts: all
-    pre_tasks:
-      # XXX java.net.InetAddress.getLocalHost throws an exception without this
-      - shell: echo "127.0.0.1 localhost {{ ansible_hostname }}" > /etc/hosts
-        changed_when: False
-    roles:
-      - ansible-role-hadoop-namenode
-    vars:
-      hadoop_config:
-        slaves:
-          - localhost
-        core_site:
-          - 
-            - name: fs.defaultFS
-            - value: 'hdfs://localhost/'
-          -
-            - name: hadoop.tmp.dir
-            - value: /var/db/hadoop
-          -
-            - name: io.file.buffer.size
-            - value: 131072
-        hdfs_site:
-          -
-            - name: dfs.namenode.name.dir
-            - value: "${hadoop.tmp.dir}/dfs/name"
-          - 
-            - name: dfs.blocksize
-            - value: 268435456
-          -
-            - name: dfs.namenode.handler.count 
-            - value: 100
-          -
-            - name: dfs.datanode.data.dir
-            - value: "${hadoop.tmp.dir}/dfs/data"
-        yarn_site:
-          -
-            - name: yarn.resourcemanager.scheduler.class
-            - value: org.apache.hadoop.yarn.server.resourcemanager.scheduler.capacity.CapacityScheduler
-          -
-            - name: yarn.acl.enable
-            - value: false
-          -
-            - name: yarn.admin.acl
-            - value: "*"
-          -
-            - name: yarn.log-aggregation-enable
-            - value: false
-          -
-            - name: yarn.resourcemanager.hostname
-            - value: localhost
-          -
-            - name: yarn.scheduler.capacity.root.queues
-            - value: default
-          -
-            - name: yarn.scheduler.capacity.root.default.capacity
-            - value: 100
-          -
-            - name: yarn.scheduler.capacity.root.default.state
-            - value: RUNNING
-          -
-            - name: yarn.scheduler.capacity.root.default.user-limit-factor
-            - value: 1
-          -
-            - name: yarn.scheduler.capacity.root.default.maximum-capacity
-            - value: 100
-
-        mapred_site:
-          -
-            - name: mapreduce.framework.name
-            - value: yarn
-          -
-            - name: mapreduce.map.java.opts
-            - value: "-Xmx512M"
-
-License
--------
-
-BSD
-
-Author Information
-------------------
+| `hadoop_namenode_user` | user name of `namenode` | `{{ __hadoop_namenode_user }}` |
```

Comment:
it is rather difficult to find a variable among many variables as the order looks random.
is it a historical order or is there any rule?